cli/packages/prisma-cli/package.json

```diff
@@ -48,13 +50,14 @@
   "scripts": {
     "build": "rimraf dist && tsc -d && chmod +x dist/index.js",
     "lint": "tslint src/**/*.ts",
-    "publish-homebrew": "dotenv -e scripts/.env ts-node ./scripts/release-brew.ts",
-    "make-binary": "npm run build && npm run package",
+    "publish-binaries": "dotenv -e scripts/.env ts-node ./scripts/release-binaries.ts",
```

Comment:
Make separate commands for publishing binaries, it should be possible to release one without the other as we might want to (say) have beta release on brew but not curl.